### ISSUE TITLE
Keras Mask RCNN: Fatal error: CropAndResize is not a registered function/op

### ISSUE THREAD

**Issue by qizhen816** [microsoft/onnxruntime#3625]
I followed the Mask RCNN convert instruction from [https://github.com/onnx/keras-onnx/tree/master/applications/mask_rcnn](https://github.com/onnx/keras-onnx/tree/master/applications/mask_rcnn), and copied `convert_tf_crop_and_resize` function from test_utils, converting seems fine with warnings:
 ```
The layer <keras.layers.core.Lambda object at 0x000002DAD8574278> doesn't have a specific converter, fall back.
The layer <keras.layers.core.Lambda object at 0x000002DAD8574278> doesn't have a specific converter, fall back.
The layer <keras.layers.core.Lambda object at 0x000002DAD8574278> doesn't have a specific converter, fall back.
The layer <keras.layers.core.Lambda object at 0x000002DAD8574278> doesn't have a specific converter, fall back.
The layer <keras.layers.core.Lambda object at 0x000002DAD8574278> doesn't have a specific converter, fall back.
The node number after optimization: 1739 -> 879
``` 
But error occurs when onnxruntime inference:
`onnxruntime.capi.onnxruntime_pybind11_state.Fail: [ONNXRuntimeError] : 1 : FAIL : Fatal error: CropAndResize is not a registered function/op` 

FYI my onnxruntime version is 1.2.0 and keras2onnx is 1.7.0

**Comment by snnn** [comment]
Please run the following script with your model and tell us the output.
```
import onnx
import sys

from onnx import ModelProto
from google.protobuf import text_format

proto = ModelProto()

# Read the existing address book.
f = open(sys.argv[1], "rb")
proto.ParseFromString(f.read())
f.close()
print(proto.opset_import)
```

**Comment by qizhen816** [comment]
The output is:
`[domain: ""
version: 11
, domain: "com.microsoft"
version: 11
]`

**Comment by skottmckay** [comment]
opset for 'com.microsoft' domain needs to be 1 not 11. 

Maybe it's an issue with the keras2onnx converter assuming the opset for onnx (empty string) and the contrib ops (com.microsoft) should be the same.

```
    if not os.path.exists(model_file_name):
        # use opset 11 or later
        set_converter('CropAndResize', convert_tf_crop_and_resize)
        oml = keras2onnx.convert_keras(model.keras_model, target_opset=11)
        onnx.save_model(oml, model_file_name)
```

**Comment by qizhen816** [comment]
Yes, it's a keras2onnx converter issue, git fixed with one of its [patch](https://github.com/onnx/keras-onnx/issues/450). Thanks.
